dataplane/worker_gc.cpp

```diff
@@ -483,12 +483,14 @@ void worker_gc_t::handle_balancer_gc()
 
 				iter.lock();
 			}
-
-			if ((uint16_t)((uint16_t)globalbase_atomic->currentTime - (uint16_t)iter.value()->timestamp_last_packet) > balancer_state_ttl)
 			{
-				const auto& real_from_base = base.globalBase->balancer_reals[iter.value()->real_unordered_id];
-				++counters[real_from_base.counter_id + (tCounterId)balancer::gc_real_counter::sessions_destroyed];
-				iter.unset_valid();
+				auto value = *iter.value();
```

Comment:
Why you copy `value` in this place? Maybe use pointer `auto* value = iter.value()`?